### DISCUSSION TITLE
CORS - ImageContentSearch project

### DISCUSSION THREAD

**Discussion by agiusalexandre** [aws-samples/aws-cdk-examples#642] [Q&A]
### The Question
I've followed the installation steps of https://github.com/aws-samples/aws-cdk-examples/tree/master/python/image-content-search.
The Web page is correctly loaded on my brwoser but when I upload an image I got some CORS issue : 

```
Failed to load resource: net::ERR_FAILED
Access to XMLHttpRequest at 'https://imagecontentsearch-icsimages9b323b5b-25gdx8b459vu.s3.amazonaws.com/' from origin 'https://m43qq8ths1.execute-api.eu-west-1.amazonaws.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

I've double checked the s3 bucket CORS definition :

```
[
    {
        "AllowedHeaders": [
            "*"
        ],
        "AllowedMethods": [
            "POST",
            "GET"
        ],
        "AllowedOrigins": [
            "*"
        ],
        "ExposeHeaders": []
    }
]
```

### Environment

  - **CDK CLI Version:** 1.111.0 (build 556ca93)
  - **Example:** ImageContentSearch
  - **Example Version:** 
  - **OS:** <!-- OSX
  - **Language:** Python

### Other information 

```
Failed to load resource: net::ERR_FAILED
Access to XMLHttpRequest at 'https://imagecontentsearch-icsimages9b323b5b-25gdx8b459vu.s3.amazonaws.com/' from origin 'https://m43qq8ths1.execute-api.eu-west-1.amazonaws.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**Comment by mohamed-ali** [comment]
Hi, 
I am facing the same issue. Is there an ETA for a fix? 
@mohsen-aws, since you are the author, could you help with this? 
Thanks.

**Comment by mohamed-ali** [comment]
Hi, 

The upload fails because the credentials that need to be passed to the form changed.

To fix the issue, edit the input parameters in the upload form following the example at the bottom of this page: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-presigned-urls.html#using-presigned-urls-to-perform-other-s3-operations.
